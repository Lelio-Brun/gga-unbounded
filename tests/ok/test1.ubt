group_setting 1.

oracle o1(m:Fp) =
  return [] in G1.

win (a:Fp, b:Fp) = ( forall j: (sum i: a_i * b_i ) - b_j = (sum j1: b_j1) + a_i7  /\
                     forall i5,j: (sum i1: a_i5*b_i1) = b_j                       /\
                     (sum i3: a_i3) = 0                                           /\
                     (sum i: a_i) - (sum j: b_j) = 0
	                	).
	                	
proof.

simplify.
