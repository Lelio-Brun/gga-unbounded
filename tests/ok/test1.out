Working on goal 1 out of 1.
  1. (sum k': a_k') = 0
  2. -(a_i7) = 0
  3. 0 = 0
  4. forall j'. b_j' = 0
